modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase

```diff
@@ -77,15 +77,34 @@ definition {
 	}
 
 	@description = "LPS-86642 - Verify that when there is no SLA defined for the process, the card should display only the Total Pending column populated"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test OnlyTotalPendingColumnIsPopulatedWhenNoSLA {
 		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
```

Comment:
Hello @AfonsoHSantos, Can you remove this `WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");`, please?

Thank you!

Reply:
fixed.